webui/src/Components/ButtonPreview.css

```
@@ -60,12 +60,12 @@
 		object-fit: contain;
 		display: block;
 		margin: 0 auto;
-		padding-bottom: calc(100% - 2px); // minus border
+		padding-bottom: calc(100% - 2px); /* minus border */
 
 		border-radius: Max(12%, 4px);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)